plugins/template-prepopulate/src/components/PrepopulateTemplateComponent/PrepopulateTemplateComponent.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Grid,
+  CardMedia
+  ,Box
+   ,Button,
+   Card ,
+   CardActions,
+   CardContent,
+   Typography, 
+   LinearProgress} from '@mui/material'
+import  LoadApiData from '../api/LoadApiData';
+
+import { useApi, configApiRef } from '@backstage/core-plugin-api';
+
+export const  handleClick =async (link:string,base:string,appurl:string) => { 
+  
+
+  const raw = JSON.stringify({
+    "url": link
+  });
+
+  const  myHeaders = new Headers();
+  myHeaders.append("Content-Type", "application/json");
+
+  var requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+
+  };
+  
+   const response= await fetch(base+"/api/prepopulatetemplate/downloadyaml",requestOptions);
+    const result = await response.text();
+    
+   const yaml = require('js-yaml');
+    const doc = yaml.load(result,'utf8');
```

Comment:
formatting and we should move 35 to the top of the file

Reply:
corrected the formatting and move 35 to the top